src/core/fair_queue.cc

```diff
@@ -429,6 +446,10 @@ void fair_queue::dispatch_requests(std::function<void(fair_queue_entry&)> cb) {
         }
     }
 
+    if (!_handles.empty() && (dispatched >= _group.per_tick_grab_threshold())) {
+        _throttled_per_tick_threshold++;
```

Comment:
So the way we would use this metric is mostly expecting it to be zero, and if it's not zero we know we are hitting the per-tick threshold, right?

That is, it's hard to know how "bad" the situation is by the value alone, right? We don't know how many total ticks there have been?

Reply:
> So the way we would use this metric is mostly expecting it to be zero, and if it's not zero we know we are hitting the per-tick threshold, right?

Yes I think any non zero value is realistically bad.

> That is, it's hard to know how "bad" the situation is by the value alone, right? We don't know how many total ticks there have been?

One reactor poll is one tick so we should be able to get a ratio of throttled ticks / poll count.